ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -123,7 +123,6 @@ chain:
           if [[ "${DISCONNECTED}" == "true" ]];
           then
             mirror_test_images
-            patch_image_streams
```

Comment:
Why do we need to remove it?

Reply:
correct, let's keep it for backward compatibility with <= 4.17